nodejs/eks/nodegroup.ts

```diff
@@ -129,6 +129,22 @@ export interface NodeGroupBaseOptions {
      */
     taints?: { [key: string]: Taint };
 
+    /**
+     * Extra args to pass to the Kubelet.  Corresponds to the options passed in the `--kubeletExtraArgs` flag to
+     * `/etc/eks/bootstrap.sh`.  An array of flags should be provided, such as: ['--port=10251', '--address=0.0.0.0'].
+     * Note that the `labels` and `taints` properties will be applied to this list after to the expicit
+     * `kubeletExtraArgs`.
+     */
+    kubeletExtraArgs?: string[];
+
+    /**
+     * Additional args to pass directly to `/etc/eks/bootstrap.sh`.  Fror details on available options, see:
+     * https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh.  Note that the `--apiserver-endpoint`,
+     * `--b64-cluster-ca` and `--kubelet-extra-args` flags are included automatically based on other configuration
+     * parameters.
+     */
+    bootstrapExtraArgs?: string;
```

Comment:
Is it worth making this an array instead of a string to collect the flags? Seems like an array would also make the string generation simpler below using a `.join`

Reply:
I think I'll actually do the opposite and make both of these just `string`.  They are meant to just pass through the raw additional args to the command line, and any interpretation of the args as an array runs some risk of being both less clear what exactly it will do, as well as being subtly incompatible with some argument scheme the underlying command expects.